### DISCUSSION TITLE
Hide taskbar on primary screen

### DISCUSSION THREAD

**Discussion by Stephanzion** [valinet/ExplorerPatcher#918] [Ideas]
I use 2 monitors and it would be very convenient to have the taskbar on the second screen only. Windows 11 allows you to display the taskbar only on the main screen, but if you make the second screen as main one, all applications will open on it, as well as a bunch of other inconveniences.

**Please add an option to hide one of the taskbars but keep the other one.**

![1](https://user-images.githubusercontent.com/34947387/155528645-7582daa5-79c7-4685-9cba-3b768e8bd77c.jpg)

_this example shows that the taskbar on the main screen is redundant_

**Comment by Hezkezl** [comment]
Replying to this one because it's the main reason I downloaded ExplorerPatcher...in hopes that it would be able to do this. On Windows 10, I could drag and drop the taskbar to any one of my 3 monitors and ONLY have it display there. Clicking the start button on my keyboard would make it pop up on that secondary monitor, and not the main monitor. 

Windows 11 completely broke that functionality, and the only option now is to display it on your Main monitor, or ALL monitors. There is no 'only display on secondary monitor' or 'only display on tertiary monitor' choice possible like there was on Windows 10.

Even if there was some kind of option to just auto-hide the taskbar on the primary monitor and/or tertiary monitor and just leave the secondary monitor one visible, that would work too. I just don't want the start menu burning in to the bottom of my 3rd monitor because it doesn't often get used, and my main monitor is often used for gaming so the taskbar rarely shows there.

Making my secondary monitor the 'Main' monitor won't work, because then games will pop up on that monitor...which is the exact opposite behavior I want :(
